go/cmd/vtctldclient/command/tablets.go

```diff
@@ -557,6 +568,26 @@ func commandStopReplication(cmd *cobra.Command, args []string) error {
 	return err
 }
 
+func commandGetPermissions(cmd *cobra.Command, args []string) error {
+	alias, err := topoproto.ParseTabletAlias(cmd.Flags().Arg(0))
+	if err != nil {
+		return err
+	}
+
+	cli.FinishedParsing(cmd)
+
+	resp, err := client.GetPermissions(commandCtx, &vtctldatapb.GetPermissionsRequest{
+		TabletAlias: alias,
+	})
+	if err != nil {
+		fmt.Printf("%v\n", err)
```

Comment:
you can just `return err`, and cobra will do the rest

Reply:
I believe you meant to remove fmt.print command in case of error...